packages/ckeditor5-list/src/listproperties.js

```diff
@@ -68,7 +68,7 @@ export default class ListProperties extends Plugin {
  * When set, the list style feature will be enabled. It allows changing the `list-style-type` HTML attribute of the lists.
```

Comment:
It's worth mentioning here that it's controlling either `list-style-type` style or the `type` attribute depending on the value.